ChangeLog

```diff
@@ -1,3 +1,4 @@
+Fix user id for blacklist when update docker
```

Comment:
Is there any issue associated?

(Ok if there isn't... just to check)

Reply:
Related issue was added and referenced in CNR